Sources/ComposableCoreLocation/Internal/Deprecations.swift

```diff
@@ -2,7 +2,17 @@
 
 #if DEBUG
   extension LocationManager {
-    @available(*, deprecated, renamed: "failing")
+    @available(*, deprecated, message: "Use 'Effect.cancellable' and 'Effect.cancel' to manage the lifecycle of 'LocationManager.delegate'")
+    public func create(id: AnyHashable) -> Effect<Action, Never> {
+      self.delegate().cancellable(id: id)
+    }
+
+    @available(*, deprecated, message: "Use 'Effect.cancellable' and 'Effect.cancel' to manage the lifecycle of 'LocationManager.delegate'")
+    public func destroy(id: AnyHashable) -> Effect<Never, Never> {
+      .cancel(id: id)
+    }
+
+    @available(*, unavailable, message: "Use 'LocationManager.failing', instead")
```

Comment:
Making this unavailable since there's not really a reasonable way to maintain a mapping...